metacat-main/src/main/java/com/netflix/metacat/main/configs/MetricsConfig.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.main.configs;
+
+import com.netflix.spectator.api.DefaultRegistry;
+import com.netflix.spectator.api.Registry;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configuration of Metrics.
+ *
+ * @author rveeramacheneni
+ */
+@Configuration
+public class MetricsConfig {
+
+    /**
+     * A default registry.
+     *
+     * @return The registry to use.
+     */
+    @Bean
+    @ConditionalOnMissingBean(Registry.class)
+    public Registry spectatorRegistry() {
+        return new DefaultRegistry();
```

Comment:
You won't want this when we pull things internal. I'd suggest maybe setting a `@ConditionalOnProperty` to disable this more explicitly. The missing bean condition is probably going to get evaluated out of order relative to our internal auto configs, so that might not get it done.

Reply:
Makes sense. Added a `@ConditionalOnProperty`